kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx

```diff
@@ -55,17 +51,23 @@ const ActionsCell: React.FC<CellContext<Topic, unknown>> = ({ row }) => {
           with DELETE policy
         </DropdownItemHint>
       </ActionDropdownItem>
-      <DropdownItem
+      <ActionDropdownItem
+        disabled={!isTopicDeletionAllowed}
```

Comment:
do we have to disable it like this cause the old one did not have this disable prop, but the Permission parts are correctly done.

Reply:
I thought it was a permission bug like the one in this picture.
![image](https://user-images.githubusercontent.com/7091756/225810967-01e59a41-f50e-4e1e-b42d-e9273f596017.png)

Reply:
@Mgrdich Is there anything else that needs to be modified?